Integrations/GoogleBigQuery/GoogleBigQuery.py

```diff
@@ -0,0 +1,192 @@
+    query_job_config = bigquery.QueryJobConfig()
+    if allow_large_results:
+        query_job_config.allow_large_results = str_to_bool(allow_large_results)
+    if default_dataset_string:
+        query_job_config.default_dataset = default_dataset_string
+    if destination_table:
+        query_job_config.destination = destination_table # must be in the format your-project.your_dataset.your_table
+    if kms_key_name:
+        query_job_config.destination_encryption_configuration = bigquery.table.EncryptionConfiguration(kms_key_name) # make sure this shouldn't be just kms_key_name
+    if dry_run:
+        query_job_config.dry_run = str_to_bool(dry_run)
+    if use_legacy_sql:
+        query_job_config.use_legacy_sql = str_to_bool(use_legacy_sql)
+    if use_query_cache:
+        query_job_config.use_query_cache = str_to_bool(use_query_cache)
+    if priority:
+        query_job_config.priority = priority
+    if write_disposition:
+        query_job_config.write_disposition = write_disposition
+
+    return query_job_config
+
+
+''' COMMANDS + REQUESTS FUNCTIONS '''
+
+def query(query_string, project_id, location, allow_large_results, default_dataset, destination, kms_key_name, dry_run, priority, use_query_cache, use_legacy_sql,
+          google_service_creds, job_id, write_disposition):
+    bigquery_client = start_and_return_bigquery_client(google_service_creds)
+    job_config = build_query_job_config(allow_large_results, default_dataset, destination, dry_run, priority, use_query_cache, use_legacy_sql, kms_key_name, write_disposition)
+    # not sure if query should be a keyword arg
+    query_job = bigquery_client.query(query = query_string, job_config = job_config, location = location, job_id = job_id, project = project_id)
+    if not (dry_run and str_to_bool(dry_run)):
+        query_results = query_job.result()
+        return query_results
+    # if dry run is activated, the results (number of bytes the query will process) are returned in the job itself
+    else:
+        return query_job
+
+
+def query_command():
+    args = demisto.args()
+    query_results = query(args['query'], args.get('project_id', None), args.get('location', None), args.get('allow_large_results', None),
+                          args.get('default_dataset', None), args.get('destination_table', None), args.get('kms_key_name', None), args.get('dry_run', None),
+                          args.get('priority', None), args.get('use_query_cache', None), args.get('use_legacy_sql', None),
+                          demisto.params()['google_service_creds'], args.get('job_id', None), args.get('write_disposition', None))
+
+    context = {}
+    contents = []
+    rows_contexts = []
+    human_readable = 'No results found.'
+    dry_run = args.get('dry_run', None)
+    if dry_run and str_to_bool(dry_run):
+        human_readable = '### Dry run results: \n This query will process {0} bytes'.format(query_results.total_bytes_processed)
```

Comment:
this is all the data for dry run?

Reply:
Somewhere I saw another field returned, but:
A. I can't seem to find that documentation after searching for it. Only this data is found.
B. Every example, including in the Google documentation, only uses this data as this is the significant data which will allow you to estimate the cost of the query. This is what users expect to see, which is the key factor in human readables to my understanding.